src/scripts/timezone.js

```diff
@@ -103,6 +104,10 @@ module.exports = (app) => {
       return;
     }
 
+    // Don't increment stats until we're here, to make sure the bot is actually
+    // doing something.
+    incrementStats("Baby Tock");
```

Comment:
[question] does baby tock get called automatically? i'm wondering if it should be included in the stats count.

Reply:
That's a good point! It is auto-triggered and probably never on purpose (except as a joking exercise). Inclusion bot is in a similar boat, but I think there's potential value in knowing how often that triggers. So I'll take out the stats on baby tock, but do you think leaving them on for inclusion bot is still sensible?

Reply:
Great point about inclusion bot. I do think it's valuable to have those stats!